src/App.css

```diff
@@ -61,6 +61,44 @@
    vertical-align: middle;
 }
 
+.delete-button {
+  margin-left: 10px;
+  width:  48px;
+  height: 48px; 
+  display: flex;
+  border-radius: 10px;
```

Comment:
The border of this element should also have the color red and the background shouldn't be gray
![image](https://user-images.githubusercontent.com/37349324/211181000-a0b4df5c-6cc3-4dda-bd32-e95286c8daf9.png)